### DISCUSSION TITLE
babel plugin next-page-config not work with babel plugin istanbul

### DISCUSSION THREAD

**Discussion by nikitosss** [vercel/next.js#12521] [Help]
# Bug report

## Describe the bug

I am add in my .babelrc plugin istanbul for Cypress coverage
Project has amp pages
Error while building project with message: Invalid page config export found. Expected object but got SequenceExpression

## System information

- OS: Windows
- Browser chrome
- Version of Next.js: 9.3.6
- Version of Node.js: 12.16.3

## Additional context

To reproduce you can clone the example https://github.com/bahmutov/next-and-cypress-example
and add amp support for the page

**Comment by timneutkens** [comment]
Feel free to investigate and fix the issue.

**Comment by einSelbst** [comment]
So somehow the instrumented page config looks invalid to next.js.
Not a solution but a workaround: Ignore the page config from instrumenting by istanbul, see https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md

```
/* istanbul ignore next */
export const config = {
  amp: 'hybrid',
}
```
